uiowa/uiowa
# i3485 - Added GAC pdf print link to bottom of aos view

Closes #3485 


# How to test


1. Checkout branch :) 
2. blt ds --site=admissions.uiowa.edu
3. drush @admissions.local uli
4. go to https://admissions.local.drupal.uiowa.edu/areas-of-study/print-links
5. check to see that the Global Assessment Certificate Program (GAC) link is there and works
6. Badda bing bada boom yay - approve :)
